test/hotspot/gtest/runtime/test_stubRoutines.cpp

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 1997, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Most of this file is copied from `stubRoutines.cpp`, hence I continue to use the copyright year from `stubRoutines.cpp`. 

But I'm not 100% sure if it is correct. Please let me know if it is not. Thanks.